src/server.ts

```diff
@@ -230,11 +242,13 @@ function runPipeline(response, diagnostics, packageAggregator, diagnosticFilePat
 }
 
 /* Slice payload in each chunk size of @batchSize */
-function slicePayload(payload, batchSize, ecosystem): any {
+function slicePayload(payload, batchSize, ecosystem, manifestHash, requestId): any {
     let reqData = [];
     for (let i = 0; i < payload.length; i += batchSize) {
         reqData.push({
             "ecosystem": ecosystem,
+            "manifest_hash": manifestHash,
+            "request_id":requestId,

```

Comment:
Why this is sent as request payload? extra query params shall be passed as argument to `fetchVulnerabilities`